tracim/tracim/config/app_cfg.py

```diff
@@ -116,24 +126,25 @@ def start_daemons(manager: DaemonsManager):
     if cfg.EMAIL_PROCESSING_MODE == CFG.CST.ASYNC:
         manager.run('mail_sender', MailSenderDaemon)
 
-environment_loaded.register(lambda: start_daemons(daemons))
-interrupt_manager = InterruptManager(os.getpid(), daemons_manager=daemons)
 
-# Note: here are fake translatable strings that allow to translate messages for reset password email content
-duplicated_email_subject = l_('Password reset request')
-duplicated_email_body = l_('''
-We've received a request to reset the password for this account.
-Please click this link to reset your password:
+def configure_depot():
+    """Configure Depot."""
+    depot_storage_name = 'default'
+    depot_storage_path = CFG.get_instance().DEPOT_STORAGE_DIR
+    depot_storage_settings = {'depot.storage_path': depot_storage_path}
+    DepotManager.configure(depot_storage_name,
+                           depot_storage_settings)
```

Comment:
The standard indentation we use is with no parameter at parenthesis level : 
```
    DepotManager.configure(
        depot_storage_name,
        depot_storage_settings
    )
```

This gives a presentation robust to refactoring